ReactWindows/ReactNative/Views/Image/ReactImageManager.cs

```diff
@@ -161,15 +161,53 @@ public void SetSource(Border view, JArray sources)
             }
         }
 
+        /// <summary>
+        /// Enum values correspond to positions of prop names in ReactPropGroup attribute
+        /// applied to <see cref="SetBorderRadius(Border, int, double)"/>
+        /// </summary>
+        private enum Radius
```

Comment:
BEGIN NITPICK

In general I don't know why we add private enums when ints work just fine.

END NITPICK